modules/admin-ui/src/main/webapp/scripts/modules/events/subresources/controllers/toolsController.js

```diff
@@ -169,14 +169,14 @@ angular.module('adminNg.controllers')
     $scope.player = {};
     $scope.video  = ToolsResource.get({ id: $scope.id, tool: 'editor' });
 
-    $scope.activeTransaction = false;
+    $scope.activeRequest = false;
 
     $scope.submit = function () {
-      $scope.activeTransaction = true;
+      $scope.activeRequest = true;
       $scope.video.thumbnail.loading = $scope.video.thumbnail && $scope.video.thumbnail.type &&
               ($scope.video.thumbnail.type === 'DEFAULT');
       $scope.video.$save({ id: $scope.id, tool: $scope.tab }, function (response) {
-        $scope.activeTransaction = false;
+        $scope.activeRequest = false;
         if ($scope.video.workflow) {
```

Comment:
In case the default thumbnail changes because the user cuts the video, $scope.video.$save will overwrite $scope.video.workflow which causes this logic to fail.

Something like:

```
var closeVideoEditor = $scope.video.workflow;
$scope.video.$save({ id: $scope.id, tool: $scope.tab }, function (response) {
        $scope.activeRequest = false;
        if (closeVideoEditor) {
         ....


makes this work correctly.
```